src/interfaces/IMetaMorpho.sol

```diff
@@ -35,7 +36,21 @@ struct MarketAllocation {
     uint256 shares;
 }
 
-interface IMetaMorpho is IERC4626 {
+interface IMultiCall {
```

Comment:
```suggestion
interface IMulticall {
```

Reply:
It seems this casing is used both for the `multicall` function and the `Multicall` contract

Reply:
Nice one, you have lynx eyes :eyes: